students/jeongseoklee/westagram/users/views.py

```diff
@@ -1,3 +1,40 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+import re
+
+from django.http  import JsonResponse
+from django.views import View
+
+from .models      import User
+
+
+class SignUpView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+
+        try:
+            email = data['email']
+            password = data['password']
+
+            REGEX_EMAIL    = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+            REGEX_PASSWORD = '^(?=.*[a-zA-Z])(?=.*[!@#$%^*+=-])(?=.*[0-9]).{8,}$'
+
+            if not re.match(REGEX_EMAIL, email):
+                return JsonResponse({"message" : "INVALID EMAIL"}, status=400)
+
+            if not re.match(REGEX_PASSWORD, password):
+                return JsonResponse({"message" : "INVALID PASSWORD"}, status=400)
+
+            if User.objects.filter(email=email).exists():
+                return JsonResponse({"message" : "EXISTED EMAIL"}, status=400)
+
+            User.objects.create(
+                email = email,
+                password = password,
+                mobile = data['mobile'],
+                username = data['username'],
+            )
```

Comment:
= 기준으로 정렬해 주세요.

해당 컨벤션은 다른 곳에서는 하지 않는 컨밴션일 수는 있지만, 현재 소속된 곳의 컨벤션을 지키는 것은 생각보다 중요합니다. 
```suggestion
            User.objects.create(
                email    = email,
                password = password,
                mobile   = data['mobile'],
                username = data['username'],
            )
```

Reply:
컨벤션 적용 후 다시 푸시 완료하였습니다.